### ISSUE TITLE
packagecloud installation of latest on ubuntu/trusty yields 1.1.1 not 1.1.2

### ISSUE THREAD

**Issue by javabrett** [github/git-lfs#1138]
This is further to discussion in #1074.

Currently, if you have:

```
$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.3 LTS"
```
... and you:
```
$ sudo apt-get install git
$ curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | \
  sudo bash
$ sudo apt-get install git-lfs
```
... you get
```
$ git lfs version
git-lfs/1.1.1 (GitHub; linux amd64; go 1.5.3)
```

... so 1.1.1 not 1.1.2.  This is because there are no packages pushed for 1.1.2 to packagecloud, but there was for 1.1.1, and their default repo-install script detects the specific distribution unless you tell it to use an upstream.

**Comment by javabrett** [comment]
This is now fixed with the release of 1.2.0.  On packagecloud/apt you will be automatically offered an upgrade from 1.1.1 to 1.2.0.  Thanks!

**Comment by technoweenie** [comment]
Thanks for reporting. I meant to fix this, but I need to update `script/packagecloud` so it can push a subset of packages. I'd like to come back around and fix this gap too.
